EIPS/eip-1.md

```diff
@@ -356,6 +356,22 @@ Permitted Yellow Paper URLs must anchor to a specific commit, and so must match
 ^(https://github\.com/ethereum/yellowpaper/blob/[0-9a-f]{40}/paper\.pdf)$
 ```
 
```

Comment:
Please don’t modify EIP-1 in this PR.

Reply:
Ok, so I will open a new PR to add PQClean permited external source @abcoathup